src/Hl7.Fhir.STU3.Tests/Rest/FhirClientTests.cs

```diff
@@ -1324,7 +1327,7 @@ private static void testHandlingHtmlErrorStatus(BaseFhirClient client)
         {
             try
             {
-                var pat = client.Read<Patient>("Patient/1");
```

Comment:
Not a fan of this change, if it's to make a compiler warning go away, add in a Diagnostics.Trace.WriteLine($"{pat.Id}"); or similar.
This is too useful just to put a breakpoint on what's come back from the server while diagnosing things.
Having it thrown away makes it harder to inspect things.

Reply:
Yes, it was just to clean up the warnings. I'll adapt this change to your suggestion.